app/views/citizens/restrictions/_form.html.erb

```diff
@@ -6,7 +6,7 @@
     <div class="govuk-checkboxes">
       <%= form.collection_check_boxes :restriction_ids, Restriction.all, :id, :label_name do |restriction_form| %>
         <div class="govuk-checkboxes__item">
-          <%= restriction_form.check_box(class: 'govuk-checkboxes__input') %>
+          <%= restriction_form.check_box(class: 'govuk-checkboxes__input', type: 'checkbox') %>
```

Comment:
why do we need this?
`restriction_form.check_box` seems to already be generating an input of type `checkbox` without this extra parameter.